google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/QueryStageTest.java

```diff
@@ -178,7 +183,9 @@ private void compareQueryStage(QueryStage expected, QueryStage value) {
     assertEquals(expected.getWriteMsMax(), expected.getWriteMsMax());
     assertEquals(expected.getWriteRatioAvg(), value.getWriteRatioAvg(), 0);
     assertEquals(expected.getWriteRatioMax(), value.getWriteRatioMax(), 0);
+    assertEquals(expected.getSlotMs(), value.getSlotMs());
     assertEquals(expected.hashCode(), value.hashCode());
+    assertEquals(expected.toString(), value.toString());
```

Comment:
I don't understand what testEquals() unit test case is testing, You are passing the same instance for expected and value. They are the same object then all of their value will be the same.

Can you check if this is by mistake or what?

Reply:
added because of code coverage.

Reply:
Sorry, I wasn't clear on my comments.

I meant why is [this test case ](https://github.com/pmakani/java-bigquery/blob/98430ac197e689680cd3a45593fafb31383af106/google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/QueryStageTest.java#L150-L155)send same object for expected as well as actual position. I believe this must be by mistake.